examples/keycloak/README.md

```diff
@@ -0,0 +1,270 @@
+NAME           TYPE       CLUSTER-IP     EXTERNAL-IP   PORT(S)          AGE
+tsi-keycloak   NodePort   10.106.55.60   <none>        9090:32548/TCP   3h15m
+```
+In this example the value is `http://10.106.55.60:9090`, and that's what we will
+add to the application configuration deployment.
+
+</details>
+
+
+To access Keyclaok remotely in `IKS`, setup ingress access.
+<details><summary>Click to view IKS steps</summary>
+
+Obtain the ingress name using `ibmcloud` cli:
+```console
+$ # first obtain the cluster name:
+$ ibmcloud ks clusters
+$ # then use the cluster name to get the Ingress info:
+$ ibmcloud ks cluster-get --cluster <cluster_name> | grep Ingress
+```
+Build an ingress file from `example/keycloak/ingress-IKS.template.yaml`,
+using the `Ingress Subdomain` information obtained above. You can use any arbitrary
+prefix in addition to the Ingress value. For example:
+`host: tsi-keycloak.my-tsi-cluster-8abee0d19746a818fd9d58aa25c34ecfe-0000.eu-de.containers.appdomain.cloud`
+
+```yaml
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: keycloak-ingress
+spec:
+  rules:
+    # provide the actual Ingress for `host` value:
+    # use the following command to get the subdomain:
+    #    ibmcloud ks cluster get --cluster <cluster-name> | grep Ingress
+    # any prefix can be defined as a result (e.g.):
+    # - host: tsi-keycloak-v001.tsi-fra02-5240a746a818fd9d58aa25c34ecfe-0000.eu-de.containers.appdomain.cloud
+    # provide the actual Ingress for `host` value:
+  - host: tsi-keycloak.my-tsi-cluster-8abee0d19746a818fd9d58aa25c34ecfe-0000.eu-de.containers.appdomain.cloud
+    http:
+      paths:
+      - backend:
+          serviceName: tsi-keycloak
+          servicePort: 9090
+        path: /
+```
+
+create ingress:
+```console
+$ kubectl -n tsi-keycloak create -f ingress-IKS.yaml
+```
+</details>
+
+To access Keycloak on OpenShift (including IKS ROKS)
+<details><summary>Click to view OpenShift steps</summary>
+
+This assumes the OpenShift command line is already installed. Otherwise see
+the [documentation](https://docs.openshift.com/container-platform/4.2/cli_reference/openshift_cli/getting-started-cli.html)
+and you can get `oc` cli from https://mirror.openshift.com/pub/openshift-v4/clients/oc/4.3/
+
+```console
+oc -n tsi-keycloak expose svc/tsi-keycloak
+export KEYCLOAK_ADDR="http://$(oc -n tsi-keycloak get route tsi-keycloak -o jsonpath='{.spec.host}')"
+```
+
+</details>
+
+Keycloak Configuration steps:
+* Using the Keycloak URL obtained above, connect to the server with a browser and
+login using admin userid and password, as specified in the [/examples/keycloak/keycloak.yaml](/examples/keycloak/keycloak.yaml)
+* In upper left corner, below Keycloak logo, click on `Master` and select `Add realm`
+* Select `Import` and navigate to [/examples/keycloak/tsi-realm-export.json](/examples/keycloak/tsi-realm-export.json) file. Press Create.
+
+This should instantiate a new `Tsi-realm` realm that we would use for the demo.
```

Comment:
```suggestion
This should instantiate a new `tsi-realm` realm that we would use for the demo.
```

not sure if caps sensitive, but just in case.

Reply:
Actually, on UI, it shows as "Tsi-realm" so I will keep it as is.